Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -193,31 +193,27 @@ private function mapIncludeFilter(StoreInterface $store, IncludeFilters &$includ
                 $value = (bool)$item->getData('value');
                 break;
             default:
-                $this->logger->debugWithSource(
+                $this->debugWithSource(
                     sprintf(
```

Comment:
Why is sprintf used here?

Reply:
Should it be switched to a string concatenation?